lib/pcsclite.js

```diff
@@ -12,7 +11,7 @@ var parse_readers_string = function(readers_str) {
     var pos;
     var readers = [];
     var ini = 0;
-    while ((pos = bt.indexOf(readers_str.slice(ini), '\0')) > 0) {
+    while ((pos = new Buffer(readers_str.slice(ini)).indexOf("\0")) > 0) {
```

Comment:
@santigimeno take a look at my solution here https://github.com/pokusew/node-pcsclite/blob/master/lib/pcsclite.js#L11
What do you think? Isn't it better (more readable and reasonable)?